Core/include/Acts/Vertexing/AdaptiveMultiVertexFinder.ipp

```diff
@@ -553,8 +553,8 @@ auto Acts::AdaptiveMultiVertexFinder<vfitter_t, sfinder_t>::deleteLastVertex(
     fitterState.vertexCollection = oldFitterState.vertexCollection;
     fitterState.annealingState = oldFitterState.annealingState;
     fitterState.vtxInfoMap.clear();
-    for (const auto& vtx : allVerticesPtr) {
-      fitterState.vtxInfoMap.emplace(vtx, oldFitterState.vtxInfoMap[vtx]);
+    for (const auto& v : allVerticesPtr) {
+      fitterState.vtxInfoMap.emplace(v, oldFitterState.vtxInfoMap[v]);
```

Comment:
Why this?

Reply:
Because `vtx` was shadowing a variable that was previously defined (quite a bit before actually).

Reply:
ok